sdk/identity/azure_identity/CHANGELOG.md

```diff
@@ -21,12 +21,6 @@
 - Removed `old_azure_cli` feature. `AzureCliCredential` now requires a recent version of the Azure CLI (2.54.0 or later).
 - Replaced `AppServiceManagedIdentityCredential`, `VirtualMachineManagedIdentityCredential`, and `ImdsId` with `ManagedIdentityCredential` and `UserAssignedId`
 
-### Bugs Fixed
-
-- Handle MSI token servers which respond with `expires_on: i64` instead of `expires_on: String`
```

Comment:
Is there a reason this change won't be included in the release?
I'd like to get off my github fork back onto the public crate.
@heaths @chlowell

Reply:
The code is still included, I'm just removing the changelog entry because this wasn't a bug and, so far as I know, the code will have no effect because only Service Fabric returns `expires_on` as an unquoted integer (and your change isn't sufficient to add support for Service Fabric)